Package.swift

```diff
@@ -1,9 +1,15 @@
 import PackageDescription
 
 let package = Package(
-  name: "SnowMan",
-  dependencies: [
-    .Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,7))
-  ]
+  	name: "SnowMan",
+  	targets: [
+  		Target(name: "Dictionary"),
+  		Target(name: "SnowManLogic", dependencies: ["Dictionary"]),
+ 		Target(name: "Application", dependencies: ["SnowManLogic"]),
+ 		Target(name: "Specs", dependencies: ["SnowManLogic"]),
+  	],
+  	dependencies: [
+    	.Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,8))
```

Comment:
I am not even kidding, this is indented on my computer. I don't know how else to do this unless I edit online. I can't see the problem before I push.